client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/commands/SubmitAsyncCommand.kt

```diff
@@ -28,16 +29,14 @@ internal class SubmitAsyncCommand(
     private val input by option("-i", "--input", help = INPUT_HELP).file(exists = true).required()
     private val attached by option("-a", "--attached", help = ATTACHED_HELP)
     private val preferredSources by option("-ps", "--preferredSources", help = PREFERRED_SOURCES)
+    private val storageMode by option("-sm", "--storageMode", help = PREFERRED_SOURCES)
```

Comment:
This one has the incorrect help, we should add one for the storage mode

We should also add this field to the documentation here https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/docs/api/submissions/operations/Submit.yaml#L48

And update others if necessary